stage #1/module #4. Collections/lesson #1. Overview/overview.md

```diff
@@ -275,9 +280,10 @@ Collections.reverse(list);
 ```
 After executing the above code, the sequence of the elements in the List will be three, two, one .
 * **shuffle()**
-The Collections _shuffle()_ method can shuffle the elements of a List.
+
+The Collections ``shuffle()`` method can shuffle the elements of a List.
```

Comment:
Can you wrap List with ``? Because everywhere you wrapped this word, but here you missed it. And line #296